### ISSUE TITLE
Reduce number of internal Cypress network requests

### ISSUE THREAD

**Issue by brian-mann** [cypress-io/cypress#606]
## Current
Currently Cypress sends up to **18** network requests for internal traffic. This is obtrusive to users, mixes with their own traffic, and reduces performance.

All of these requests come from Cypress itself.

![screen shot 2017-08-26 at 2 17 54 pm](https://user-images.githubusercontent.com/1268976/29743991-858f64f4-8a69-11e7-83fe-ea7dadf47170.png)

## Proposal
- Consolidate as many requests as possible
- Namespace these so they are more clearly identified as internal cypress network requests.

**Comment by brian-mann** [comment]
We were able to reduce the network requests by 50% from 18 to 9.

We consolidated a bunch of JS, CSS, icons, and set socket.io to websocket traffic only so it does less thrashing about.

It's probably possible to golf this down a bit by consolidating the fonts.

![screen shot 2017-08-26 at 2 20 18 pm](https://user-images.githubusercontent.com/1268976/29744015-feef6380-8a69-11e7-99d6-34a527873834.png)
